app/controllers/analyses_controller.rb

```diff
@@ -32,12 +32,26 @@ def show
     @analysis = current_user.analyses.find(params[:id])
 
     respond_to do |format|
-      format.json { render json: @analysis }
+      format.json do
+        if @analysis.gwas? && @analysis.success?
+          results = Analysis::Gwas::ManhattanPlot.new(@analysis).call.fetch(:traits).map do |trait_name, mutations, _|
+            mutations.map { |m| m[1] = "%.4f" % m[1]; m << trait_name }
```

Comment:
I am appending trait name to each mutation in the list (this is content for the results table).